[Automation] java run fails for com.fasterxml.jackson.module:jackson-module-paranamer:2.19.0

## java run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25310286762/job/74195600181
Last 300 lines of the log:
```console
Testing library: com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :jar
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

3 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 5 executed, 10 up-to-date

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.19.0 at 2026-05-04T09:15:26Z with exit code 1 after 5s
❌ ERROR: 'native-image run' for 2.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean test -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.19.0 at 2026-05-04T09:15:26Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean compileTestJava -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 774ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 894ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.19.0 at 2026-05-04T09:15:29Z with exit code 0 after 3s
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage java run
========================================================================================
Starting stage 'java run' for 2.19.0 at 2026-05-04T09:15:29Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 813ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

3 tests completed, 1 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 2.19.0 at 2026-05-04T09:15:33Z with exit code 1 after 4s
❌ ERROR: 'java run' for 2.19.0 failed with exit code 1.
FAILED[java run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
```

## java run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25310286762/job/74195600092
Last 300 lines of the log:
```console
Testing library: com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

3 tests completed, 1 failed

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 5 executed, 10 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.19.0 at 2026-05-04T09:15:08Z with exit code 1 after 5s
❌ ERROR: 'native-image run' for 2.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean test -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.19.0 at 2026-05-04T09:15:08Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean compileTestJava -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 744ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 858ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.19.0 at 2026-05-04T09:15:10Z with exit code 0 after 2s
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage java run
========================================================================================
Starting stage 'java run' for 2.19.0 at 2026-05-04T09:15:10Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 741ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :processTestResources
> Task :compileTestJava
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

3 tests completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s
14 actionable tasks: 4 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 2.19.0 at 2026-05-04T09:15:14Z with exit code 1 after 4s
❌ ERROR: 'java run' for 2.19.0 failed with exit code 1.
FAILED[java run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
```

## java run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25310286762/job/74195600083
Last 300 lines of the log:
```console
> Task :tckTest
====================
Testing library: com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :jar
> Task :checkTestTimeoutAnnotations
> Task :processTestResources
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

3 tests completed, 1 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
15 actionable tasks: 5 executed, 10 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.19.0 at 2026-05-04T09:13:16Z with exit code 1 after 4s
❌ ERROR: 'native-image run' for 2.19.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.19.0" ./gradlew clean test -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.19.0 at 2026-05-04T09:13:16Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.19.0" ./gradlew clean compileTestJava -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 624ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 691ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.19.0 at 2026-05-04T09:13:18Z with exit code 0 after 2s
========================================================================================
 com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4:2.19.0 stage java run
========================================================================================
Starting stage 'java run' for 2.19.0 at 2026-05-04T09:13:18Z
Command: GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 572ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :processTestResources
> Task :compileTestJava
> Task :testClasses

> Task :test FAILED

DefaultParanamerTest > readsParameterNamesFromEmbeddedParanamerDataField() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      []
    to contain exactly (and in same order):
      ["text", "repeatCount"]
    but could not find the following elements:
      ["text", "repeatCount"]
        at com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest.readsParameterNamesFromEmbeddedParanamerDataField(DefaultParanamerTest.java:23)

3 tests completed, 1 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.11.4/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'java run' for 2.19.0 at 2026-05-04T09:13:21Z with exit code 1 after 3s
❌ ERROR: 'java run' for 2.19.0 failed with exit code 1.
FAILED[java run][2.19.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.19.0" ./gradlew clean javaTest -Pcoordinates="com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4"]
```

--- a/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/reachability-metadata.json
+++ b/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/reachability-metadata.json
@@ -1,0 +1,22 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.module.paranamer.shaded.AdaptiveParanamer"
+      },
+      "type" : "com.fasterxml.jackson.module.paranamer.shaded.DefaultParanamer",
+      "methods" : [
+        {
+          "name" : "<init>",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.module.paranamer.shaded.AdaptiveParanamer"
+      },
+      "type" : "java.lang.StackTraceElement[]"
+    }
+  ]
+}

--- a/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/index.json
+++ b/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/index.json
@@ -5,6 +5,11 @@
     "tested-versions" : [
       "2.19.0"
     ],
+    "source-code-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/module/jackson-module-paranamer/$version$/jackson-module-paranamer-$version$-sources.jar",
+    "repository-url" : "https://github.com/FasterXML/jackson-modules-base",
+    "test-code-url" : "https://github.com/FasterXML/jackson-modules-base/tree/jackson-modules-base-$version$/paranamer/src/test/java",
+    "documentation-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/module/jackson-module-paranamer/$version$/jackson-module-paranamer-$version$-javadoc.jar",
+    "description" : "Jackson Module Paranamer adds annotation introspection support that lets Jackson discover constructor and factory method parameter names through the Paranamer library. It helps Jackson databind deserialize types whose creators rely on parameter names when explicit Jackson annotations are not present.",
     "allowed-packages" : [
       "com.fasterxml.jackson.module.paranamer"
     ]

--- a/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/index.json
+++ b/metadata/com.fasterxml.jackson.module/jackson-module-paranamer/index.json
@@ -1,6 +1,15 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "2.19.0",
+    "tested-versions" : [
+      "2.19.0"
+    ],
+    "allowed-packages" : [
+      "com.fasterxml.jackson.module.paranamer"
+    ]
+  },
+  {
     "metadata-version" : "2.11.4",
     "source-code-url" : "https://repo1.maven.org/maven2/com/fasterxml/jackson/module/jackson-module-paranamer/$version$/jackson-module-paranamer-$version$-sources.jar",
     "repository-url" : "https://github.com/FasterXML/jackson-modules-base",

--- a/stats/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/execution-metrics.json
+++ b/stats/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/execution-metrics.json
@@ -1,0 +1,111 @@
+{
+  "fix_java_run_fail:2026-05-04": {
+    "timestamp": "2026-05-04T11:36:01.508818Z",
+    "library": "com.fasterxml.jackson.module:jackson-module-paranamer:2.19.0",
+    "starting_commit": "55f69c79175c935d0d3af8eac274cf4f235efa9e",
+    "ending_commit": "89a5b5f38547c4d07807b088c6e54a73199dc4c3",
+    "previous_library": "com.fasterxml.jackson.module:jackson-module-paranamer:2.11.4",
+    "strategy_name": "java_run_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "2.19.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 4,
+            "totalCalls": 4
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 6,
+        "totalCalls": 6
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 1415,
+          "missed": 1459,
+          "ratio": 0.492345,
+          "total": 2874
+        },
+        "line": {
+          "covered": 331,
+          "missed": 359,
+          "ratio": 0.47971,
+          "total": 690
+        },
+        "method": {
+          "covered": 45,
+          "missed": 71,
+          "ratio": 0.387931,
+          "total": 116
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "2.11.4",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          },
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 4,
+        "totalCalls": 4
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 1318,
+          "missed": 1541,
+          "ratio": 0.461,
+          "total": 2859
+        },
+        "line": {
+          "covered": 320,
+          "missed": 386,
+          "ratio": 0.453258,
+          "total": 706
+        },
+        "method": {
+          "covered": 38,
+          "missed": 75,
+          "ratio": 0.336283,
+          "total": 113
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 80494,
+      "cached_input_tokens_used": 928768,
+      "output_tokens_used": 11703,
+      "iterations": 3,
+      "cost_usd": 0.8155,
+      "tested_library_loc": 331,
+      "metadata_entries": 3,
+      "test_only_metadata_entries": 13,
+      "previous_library_metadata_entries": 0,
+      "previous_library_test_only_metadata_entries": 10,
+      "code_coverage_percent": 47.97,
+      "previous_library_coverage_percent": 45.33
+    },
+    "artifacts": {
+      "test_file": "tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest.java",
+      "metadata_file": "metadata/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/stats.json
+++ b/stats/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/stats.json
@@ -1,0 +1,42 @@
+{
+  "versions" : [ {
+    "version" : "2.19.0",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 4,
+          "totalCalls" : 4
+        },
+        "resources" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 2,
+          "totalCalls" : 2
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 6,
+      "totalCalls" : 6
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 1415,
+        "missed" : 1459,
+        "ratio" : 0.492345,
+        "total" : 2874
+      },
+      "line" : {
+        "covered" : 331,
+        "missed" : 359,
+        "ratio" : 0.47971,
+        "total" : 690
+      },
+      "method" : {
+        "covered" : 45,
+        "missed" : 71,
+        "ratio" : 0.387931,
+        "total" : 116
+      }
+    }
+  } ]
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/.gitignore
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/build.gradle
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "com.fasterxml.jackson.module:jackson-module-paranamer:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/gradle.properties
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = com.fasterxml.jackson.module:jackson-module-paranamer:2.19.0
+library.version = 2.19.0
+metadata.dir = com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/settings.gradle
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'com.fasterxml.jackson.module.jackson-module-paranamer_tests'

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/AdaptiveParanamerTest.java
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/AdaptiveParanamerTest.java
@@ -8,18 +8,18 @@ package com_fasterxml_jackson_module.jackson_module_paranamer;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import com.fasterxml.jackson.module.paranamer.shaded.DefaultParanamer;
+import com.fasterxml.jackson.module.paranamer.shaded.AdaptiveParanamer;
 import java.lang.reflect.Constructor;
 import org.junit.jupiter.api.Test;
 
-public class DefaultParanamerTest {
+public class AdaptiveParanamerTest {
     @Test
-    void readsParameterNamesFromReflectionMetadata() throws Exception {
-        DefaultParanamer paranamer = new DefaultParanamer();
+    void defaultConstructorIncludesReflectionBasedParameterNameLookup() throws Exception {
+        AdaptiveParanamer paranamer = new AdaptiveParanamer();
         Constructor<ConstructorParameterFixture> constructor = ConstructorParameterFixture.class.getDeclaredConstructor(
                 String.class, int.class);
 
-        String[] parameterNames = paranamer.lookupParameterNames(constructor);
+        String[] parameterNames = paranamer.lookupParameterNames(constructor, false);
 
         assertThat(parameterNames).containsExactly("text", "repeatCount");
     }

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest.java
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest.java
@@ -1,0 +1,63 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_fasterxml_jackson_module.jackson_module_paranamer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.module.paranamer.shaded.BytecodeReadingParanamer;
+import java.io.InputStream;
+import java.lang.reflect.Method;
+import org.junit.jupiter.api.Test;
+
+public class BytecodeReadingParanamerTest {
+    @Test
+    void readsParameterNamesFromAvailableClassResource() throws Exception {
+        BytecodeReadingParanamer paranamer = new BytecodeReadingParanamer();
+        Method method = ParameterNameFixture.class.getDeclaredMethod("format", String.class, int.class);
+        assertThatClassResourceIsAvailable(method.getDeclaringClass());
+
+        String[] parameterNames = paranamer.lookupParameterNames(method, false);
+
+        assertThat(parameterNames).containsExactly("message", "repeatCount");
+    }
+
+    @Test
+    void returnsEmptyNamesWhenGeneratedClassHasNoBytecodeResource() throws Exception {
+        BytecodeReadingParanamer paranamer = new BytecodeReadingParanamer();
+        ParameterizedOperation operation = value -> value.length();
+        Method method = operation.getClass().getDeclaredMethod("apply", String.class);
+        assertThat(method.getDeclaringClass()).isSameAs(operation.getClass());
+
+        String[] parameterNames = paranamer.lookupParameterNames(method, false);
+
+        assertThat(parameterNames).isEmpty();
+    }
+
+    private interface ParameterizedOperation {
+        int apply(String value);
+    }
+
+    private static void assertThatClassResourceIsAvailable(Class<?> type) throws Exception {
+        try (InputStream resource = type.getClassLoader().getResourceAsStream(classResourceName(type))) {
+            assertThat(resource).isNotNull();
+        }
+    }
+
+    private static String classResourceName(Class<?> type) {
+        return type.getName().replace('.', '/') + ".class";
+    }
+
+    private static final class ParameterNameFixture {
+        String format(String message, int repeatCount) {
+            StringBuilder result = new StringBuilder();
+            for (int index = 0; index < repeatCount; index++) {
+                result.append(message);
+            }
+            return result.toString();
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/DefaultParanamerTest.java
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/DefaultParanamerTest.java
@@ -1,0 +1,40 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_fasterxml_jackson_module.jackson_module_paranamer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.module.paranamer.shaded.DefaultParanamer;
+import java.lang.reflect.Method;
+import org.junit.jupiter.api.Test;
+
+public class DefaultParanamerTest {
+    @Test
+    void readsParameterNamesFromEmbeddedParanamerDataField() throws Exception {
+        DefaultParanamer paranamer = new DefaultParanamer();
+        Method method = MethodParameterFixture.class.getDeclaredMethod("combine", String.class, int.class);
+
+        String[] parameterNames = paranamer.lookupParameterNames(method);
+
+        assertThat(parameterNames).containsExactly("text", "repeatCount");
+    }
+
+    public static final class MethodParameterFixture {
+        public static final String __PARANAMER_DATA = """
+                v1.0
+                combine java.lang.String,int text,repeatCount
+                """;
+
+        public String combine(String text, int repeatCount) {
+            StringBuilder result = new StringBuilder();
+            for (int index = 0; index < repeatCount; index++) {
+                result.append(text);
+            }
+            return result.toString();
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/LegacyParanamerTest.java
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/java/com_fasterxml_jackson_module/jackson_module_paranamer/LegacyParanamerTest.java
@@ -1,0 +1,40 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_fasterxml_jackson_module.jackson_module_paranamer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.module.paranamer.shaded.LegacyParanamer;
+import java.lang.reflect.Method;
+import org.junit.jupiter.api.Test;
+
+public class LegacyParanamerTest {
+    @Test
+    void readsParameterNamesFromEmbeddedParanamerDataField() throws Exception {
+        LegacyParanamer paranamer = new LegacyParanamer();
+        Method method = MethodParameterFixture.class.getDeclaredMethod("combine", String.class, int.class);
+
+        String[] parameterNames = paranamer.lookupParameterNames(method);
+
+        assertThat(parameterNames).containsExactly("text", "repeatCount");
+    }
+
+    public static final class MethodParameterFixture {
+        public static final String __PARANAMER_DATA = """
+                v1.0
+                combine java.lang.String,int text,repeatCount
+                """;
+
+        public String combine(String text, int repeatCount) {
+            StringBuilder result = new StringBuilder();
+            for (int index = 0; index < repeatCount; index++) {
+                result.append(text);
+            }
+            return result.toString();
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -1,0 +1,72 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.module.paranamer.shaded.DefaultParanamer"
+      },
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$MethodParameterFixture",
+      "fields" : [
+        {
+          "name" : "__PARANAMER_DATA"
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest$ParameterNameFixture"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest"
+      },
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$MethodParameterFixture"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "type" : {
+        "lambda" : {
+          "declaringClass" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest",
+          "interfaces" : [
+            "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest$ParameterizedOperation"
+          ]
+        }
+      }
+    }
+  ],
+  "resources" : [
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "glob" : "META-INF/services/org.assertj.core.configuration.Configuration"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "glob" : "META-INF/services/org.assertj.core.presentation.Representation"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "glob" : "com/fasterxml/jackson/module/paranamer/shaded/com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest$$Lambda/0x00000000141ae470.class"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "glob" : "com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest$$Lambda/0x00000000141ae470.class"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
+      },
+      "glob" : "com_fasterxml_jackson_module/jackson_module_paranamer/BytecodeReadingParanamerTest$ParameterNameFixture.class"
+    }
+  ]
+}

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -4,10 +4,14 @@
       "condition" : {
         "typeReached" : "com.fasterxml.jackson.module.paranamer.shaded.DefaultParanamer"
       },
-      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$MethodParameterFixture",
-      "fields" : [
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$ConstructorParameterFixture",
+      "methods" : [
         {
-          "name" : "__PARANAMER_DATA"
+          "name" : "<init>",
+          "parameterTypes" : [
+            "java.lang.String",
+            "int"
+          ]
         }
       ]
     },
@@ -21,7 +25,7 @@
       "condition" : {
         "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest"
       },
-      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$MethodParameterFixture"
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.DefaultParanamerTest$ConstructorParameterFixture"
     },
     {
       "condition" : {

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -29,6 +29,26 @@
     },
     {
       "condition" : {
+        "typeReached" : "com.fasterxml.jackson.module.paranamer.shaded.LegacyParanamer"
+      },
+      "type" : "com_fasterxml_jackson_module.jackson_module_paranamer.LegacyParanamerTest$MethodParameterFixture",
+      "fields" : [
+        {
+          "name" : "__PARANAMER_DATA"
+        }
+      ],
+      "methods" : [
+        {
+          "name" : "combine",
+          "parameterTypes" : [
+            "java.lang.String",
+            "int"
+          ]
+        }
+      ]
+    },
+    {
+      "condition" : {
         "typeReached" : "com_fasterxml_jackson_module.jackson_module_paranamer.BytecodeReadingParanamerTest"
       },
       "type" : {

--- a/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/user-code-filter.json
+++ b/tests/src/com.fasterxml.jackson.module/jackson-module-paranamer/2.19.0/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "com.fasterxml.jackson.module.paranamer.**"
+    },
+    {
+      "includeClasses" : "com_fasterxml_jackson_module.jackson_module_paranamer.**"
+    }
+  ]
+}

